akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -381,8 +430,15 @@ private[remote] class ArteryTransport(_system: ExtendedActorSystem, _provider: R
 
   val (afrFileChannel, afrFlie, flightRecorder) = initializeFlightRecorder()
 
+  def createFlightRecorderEventSink(): EventSink = {
+    // FIXME there is some concurrency issue with the FlightRecorder, when shutting down.
+    //       It crashes the JVM.
+    // flightRecorder.createEventSink()
+    IgnoreEventSink
```

Comment:
Were you calling any method that accesses the event sink from multiple threads? It is not thread-safe and the only calls in this class to it were from start and shutdown as far as I remember.